app/models/user.rb

```diff
@@ -75,6 +85,35 @@ class User < ApplicationRecord
       end
     }
 
+  # NOTE(ezekg) This mirrors behavior for <= v1.5 where a user's licenses
+  #             were destroyed when the user was destroyed. With the
+  #             advent of multi-user licenses, this is no longer the
+  #             case. Instead, the license is simply disassociated
+  #             from the user.
+  #
+  # TODO(ezekg) Uncomment once we migrate licenses to multi-user licenses.
+  # TODO(ezekg) Should we batch this?
+  #
+  # after_destroy {
+  #   next unless
+  #     Current.api_version.present? # skip outside of web context
+
+  #   next unless
+  #     Semverse::Constraint.new('<= 1.5')
+  #                         .satisfies?(
+  #                           Semverse::Version.new(Current.api_version),
+  #                         )
+
+  #   destroy_association_async_job.perform_later(
+  #     owner_model_name: self.class.to_s,
+  #     owner_id: id,
+  #     association_class: _licenses.klass.to_s,
+  #     association_ids: _licenses.ids,
```

Comment:
This should only destroy the set of licenses ONLY if the set contains a single license.